cypress/unit/store-getter-getAssessmentToolJSONOutput.cy.js

```diff
@@ -0,0 +1,52 @@
+import { getters } from "~/store";
+
+let store = {
+
+    getters: getters,
+
+    state: {
+        dataDictionary:  {
+            annotated:  {
+                column1:  {
+                  Description: "Some cool description here.",
+                  missingValues: ["Missing"]
+                }
+            }
+        },
+        toolTerms: [
+            {
+                label: "MOCA",
+                identifier: "cogAtlas:MOCA",
+                selected: false
+            }
+        ],
+        columnToToolMap: {
+            column1: "cogAtlas:MOCA"
+        }
+
+    }
+};
+
+describe("getAssessmentToolJSONOutput", () => {
+
+    it("Make sure Assessment tool json output is schema compliant", () => {
+
+        // Act - Get formatted json output data for discrete value column
```

Comment:
I'm OK with adding these flags, but then we should also add it for "Assert" below. That's the main benefit to me: you see where the asserts begin